client/src/global/components/schema/Types/Issue.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import PropTypes from "prop-types";
+import lh from "helpers/linkHandler";
+import { renderOffer, renderNamesList } from "../helpers";
+import BaseSchema from "../BaseSchema";
+
+export default function Issue({ issue }) {
+  const { attributes, relationships } = issue;
+
+  const renderJournal = () => {
+    const { journal } = relationships;
+
+    return journal
+      ? {
+          "@type": "Periodical",
+          name: journal.attributes.title
+        }
+      : null;
+  };
+
+  const {
+    slug,
+    title,
+    metadata,
+    publicationDate,
+    createdAt,
+    updatedAt,
+    avatarStyles,
+    number
+  } = attributes;
+  const { creators, contributors } = relationships;
+
+  const issueData = {
+    "@type": "PublicationIssue",
+    "@id": metadata.doi,
+    name: title,
+    issueNumber: number,
+    url: lh.link("frontendProjectDetail", slug),
+    issn: metadata.issn,
```

Comment:
I didn't see that `issn` was a valid property for PublicationIssue. You may need to add it using `identifier`. Let's add ISBN and DOI at the same time.